src/schemaValidation/validateSchema.ts

```diff
@@ -0,0 +1,169 @@
+import { CliError } from "../params";
+import { ReleaseFileType } from "../types";
+import Ajv, { ErrorObject } from "ajv";
+import ajvErrors from "ajv-errors";
+// Schemas
+import manifestSchema from "./schemas/manifest.schema.json";
+import composeSchema from "./schemas/compose.schema.json";
+import setupWizardSchema from "./schemas/setup-wizard.schema.json";
+import { SetupWizard } from "../releaseFiles/setupWizard/types";
+import { Manifest } from "../releaseFiles/manifest/types";
+import { Compose } from "../releaseFiles/compose/types";
+
+const ajv = new Ajv({
+  allErrors: true,
+  coerceTypes: true,
+  strictSchema: false
+});
+
+ajvErrors(ajv);
+
+console.log("Validating manifest schema");
```

Comment:
Why are this logs here? Doesn't this log always in response to any command?

Reply:
ups, I was debugging, forgot to delete it